MTCircularSlider/Classes/MTCircularSlider.swift

```diff
@@ -36,6 +36,7 @@ public enum Attributes {
 	case trackShadowDepth(CGFloat)
 	case trackMinAngle(CGFloat)
 	case trackMaxAngle(CGFloat)
+    case trackRounding(Bool)
```

Comment:
Indentation here and below?

Reply:
`areTrackCapsRound`/`areTrackEndsRound` or something similar?

Reply:
Think areTrackCapsRound be better, cause Apple use similar naming.

https://developer.apple.com/documentation/uikit/uibezierpath/1624347-linecapstyle